_includes/js/custom.js

```diff
@@ -99,4 +69,383 @@ jQuery(document).ready(function() {
         }).debounce(100));
     };
     search();
+    //set the first character os string to uppercase
+    function capitalize(string) {
+        return string.charAt(0).toUpperCase() + string.slice(1);
+    }
+    //surround data table rows generated in rendertable function with table head
+    function tableHtml(content, domain = false, tableDomains) {
+        return `<div><strong>${ domain? capitalize(tableDomains): ""}</strong></div>
+                <table id="ont_table" class="table table-hover sortable">
+                    <thead>
+                        <tr class="row">
+                            <th class="col-sm-1">
+                                ID
+                                <button type="button" class="btn btn-outline-default btn-sm" title="Sort by ID" data-sort="id" style="float: right;">
+                                    <span aria-hidden="true" class="glyphicon glyphicon-chevron-down"></span>
+                                </button>
+                            </th>
+                            <th class="col-sm-1">
+                                Title
+                                <button type="button" class="btn btn-outline-default btn-sm" title="Sort by title" data-sort="title" style="float: right;">
+                                    <span aria-hidden="true" class="glyphicon glyphicon-chevron-down"></span>
+                                </button>
+                            </th>
+                            <th class="col-sm-3">Description</th>
+                            <th class="col-sm-4">Quick Access</th>
+                            <th class="col-sm-2">
+                                Re-Use
+                                <button type="button" class="btn btn-outline-default btn-sm" title="Sort by License" data-sort="license" style="float: right;">
```

Comment:
this button seems to be creating additional horizontal space in the license column rather, which creates a weird horizontal block throughout the table
<img width="1194" alt="Screen Shot 2022-07-31 at 14 55 28" src="https://user-images.githubusercontent.com/5069736/182027460-5dec83d8-b435-4c61-ad4c-417a35242d3e.png">

Reply:
A yeah this is also probably taking some horizontal space the other columns good make good use of.